sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -575,8 +575,7 @@ object RapidsConf {
   val IMPROVED_FLOAT_OPS = conf("spark.rapids.sql.improvedFloatOps.enabled")
     .doc("For some floating point operations spark uses one way to compute the value " +
       "and the underlying cudf implementation can use an improved algorithm. " +
-      "In some cases this can result in cudf producing an answer when spark overflows. " +
-      "Because this is not as compatible with spark, we have it disabled by default.")
+      "In some cases this can result in cudf producing an answer when spark overflows.")
     .booleanConf
     .createWithDefault(true)
```

Comment:
we intentionally put the confs into our integration test with the defaults they had, if we change the default here those tests I believe should be updated as well...

integration_tests/src/main/python/spark_session.py:
    'spark.rapids.sql.improvedFloatOps.enabled': 'false',

Can you please file a 22.08 issue to update those tests. We may want to check other defualtas as well

Reply:
created issue #5844